### PR TITLE
Enable literalMidWordUnderscores option in Showdown

### PR DESCRIPTION
**Pull request by jekunen**
This pull request align the Markdown rendering of middle of words underscores with how GitHub shows it and what users have come to expect. The Showdown live demo http://demo.showdownjs.com/ also has the same option enabled by default. Similar options changes for better output consistency have previously been done at least in #2311.

Example:
**Input text:** `string_with_underscores`\
**User expectation:** string_with_underscores\
**Release 1.7.19 result:** string**with**underscores\
**With pull request changes:** string_with_underscores

Link to showdown's literalMidWordUnderscores option documentation: https://github.com/showdownjs/showdown/wiki/Showdown-Options#literalmidwordunderscores

### CODE REVIEW

File: opengrok-web/src/main/webapp/js/utils-0.0.43.js [hunk 1]
```diff
@@ -1484,6 +1484,7 @@ function init_markdown_converter() {
                         converter.setOption('tasklists', true);
                         converter.setOption('simplifiedAutoLink', true);
                         converter.setOption('parseImgDimension', true);
+                        converter.setOption('literalMidWordUnderscores', true);
```
Comment (ahornace) [inline opengrok-web/src/main/webapp/js/utils-0.0.43.js:1487]: You need to increment the file version to `0.0.44.js` and update `Scripts.java` (look at https://github.com/oracle/opengrok/pull/3717/files for reference). The rename is needed due to caching policies.

**Comment by ahornace** [comment]
Also, to be able to accept your change, it is required for you to accept OCA conditions https://oca.opensource.oracle.com

Could you please do so or (at the very least) state that you accept the terms?

**Comment by jekunen** [comment]
I state that I accept the terms.

**Comment by ahornace** [comment]
Thanks!

**Pull request merged**